agrotoxico/models/agrotoxico.py

```diff
@@ -2,10 +2,25 @@
 
 import uuid
 
+from cultura.models.cultura import Cultura
+
 from .tipo_agrotoxico import TipoAgrotoxico
 
 
 class Agrotoxico(models.Model):
     idAgrotoxico = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     nome = models.CharField(max_length=80, unique=True)
     tipo = models.ForeignKey(TipoAgrotoxico, on_delete=models.PROTECT)
+
+    @property
+    def espera(self):
+        return self.espera_set
+
+
+class Espera(models.Model):
```

Comment:
Tabela de relacionamento, falta linkagem com a tabala de agrotóxico, não sei é essa mesma, ou se é cultura (depende da direção da seta no DLD). Utilizar o [through](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.ManyToManyField.through).

Outra documentação sobre dados extras na tabela: https://docs.djangoproject.com/en/4.0/topics/db/models/#extra-fields-on-many-to-many-relationships

Reply:
nao tem necessidade de especificar o campo manytomany aqui porque, provavelmente, nunca vai ser utilizado, pois atualmente não há a necessidade de saber informações somente da cultura relacionada. O principal é o dado que esta no relacionamento(diasCarencia) e se quiser saber da cultura, pode pegar a partir dele

Reply:
Mesmo assim, foi definido na modelagem que essa tabela seria gerada pelo relacionamento n:m, logo a gente deve implementar da forma correta. Além disso, apenas apostar que "provavelmente" não vai acontecer não garante a corretude da implementação sem o relacionamento ManyToMany, pois se isso fosse verdade, essa tabela nem existira e seria uma foreign key ou algo do tipo.

Reply:
faz sentido